server/api/impl/aws/aws_keys.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package aws
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+)
+
+// getFunctionName generates function name for a specific app
+// name can be 64 characters long.
```

Comment:
do we need to ensure (after all the steps below to generate the final name) that it is or up to 64 chars long?

Reply:
It is ensured in the code. Final name will be less than 64 chars.

Reply:
prob. would be clearer if we add a final check as opposed to trying to determine that the individual pieces are of the right size, so that when concatenated they amount to less than 64 chars.